### PR TITLE
[14.0] [IMP] sale_purchase_stock_line_note: line note in report

### PR DESCRIPTION
**Pull request by renda-dev**
If the setting is enabled, display stock picking line note in the report

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional ok! 

Added notes to stock move lines, and to report with a setting - as per CONFIGURE: 

Settings > Stock: Enable "Display line notes in picking operations report" to print line notes on report.

@grindtildeath good for you?

**Review by aayartsev — APPROVED** [review]
Looks good

File: sale_purchase_stock_line_note/models/stock_move_line.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
```
Comment (grindtildeath) [inline sale_purchase_stock_line_note/models/stock_move_line.py:3]: Can you add an header please?

**Review by grindtildeath — APPROVED** [review]
Thanks for the improvement. LGTM out of missing header, so pre approving.

File: sale_purchase_stock_line_note/models/stock_move_line.py (hunk 1 above)
Comment (renda-dev) [reply sale_purchase_stock_line_note/models/stock_move_line.py:3]: Done, let me know if there is anything else that is missing :)

**Comment by francesco-ooops** [comment]
@grindtildeath can you merge? thanks!

**Comment by francesco-ooops** [comment]
@grindtildeath gentle reminder :)

**Pull request merged**